### PR TITLE
Create ru_ru.lang

### PR DESCRIPTION
**Pull request by Blueberryy**
Enjoy :p

### CODE REVIEW

File: src/main/resources/assets/cubicchunks/lang/ru_ru.lang [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#################################
+###        general gui        ###
+#################################
+
+cubicchunks.gui.worldmenu.cc_disable=КубическиеЧанки: Нет
+cubicchunks.gui.worldmenu.cc_default=КубическиеЧанки: По умолчанию
```
Comment (Niko-sk2x) [inline src/main/resources/assets/cubicchunks/lang/ru_ru.lang]: Are you sure it's a good idea to translate what is essentially name of the mod?

**Comment by Blueberryy** [comment]
Mhm.

**Comment by Blueberryy** [comment]
Changed.

**Comment by muzikbike** [comment]
should be prime for merging in now

**Pull request merged**